fastlane/Fastfile

```diff
@@ -32,10 +32,15 @@ platform :ios do
     fail ArgumentError, "missing version" unless new_version_number
     previous_version_number = get_version_number(target: "Purchases")
     increment_version_number(version_number: new_version_number)
-    version_bump_podspec(version_number: new_version_number, path: "Purchases.podspec")
-    version_bump_podspec(version_number: new_version_number, path: "PurchasesCoreSwift.podspec")
-    increment_build_number(previous_version_number, new_version_number, '../Purchases/Misc/RCSystemInfo.m')
-    increment_build_number(previous_version_number, new_version_number, '../.jazzy.yaml')
+    files_to_update = [
+      '../PurchasesCoreSwift.podspec',
+      '../Purchases.podspec',
```

Comment:
this update is because the `Purchases.podspec` has two places where the version needs to be bumped, one is the version for itself and the other is the version of `PurchasesCoreSwift`, and `version_bump_podspec` would only update one of them.